Documentation update: Concepts (#2218)
fetchai/agents-aea: docs/app-areas.md

> The <a href="../agent-oriented-development">agent-oriented development</a> guide presented some of the fundamental attributes of the environments which AEA-based solutions are suited for.

Reviewer: ```suggestion
The <a href="../agent-oriented-development">agent-oriented development</a> guide presents some of the fundamental attributes of the environments which AEA-based solutions are suited for.
```

Reviewer: Past tense everywhere. No mixing of tenses.

Author: Done

> Under the light of that discussion, in this page we identify a number of application areas for AEA-based solutions. This list is by no means comprehensive. In fact, we are most excited about applications we have not thought of before.

Reviewer: ```suggestion
In light of that discussion, on this page we identify a number of application areas for AEA-based solutions. This list is by no means comprehensive. In fact, we are most excited about applications we have not thought of before.
```

Author: Done

## General application areas
Under the light of that discussion, in this page we identify a number of application areas for AEA-based solutions. This list is by no means comprehensive. In fact, we are most excited about applications we have not thought of before.

As described in the guide on <a href="../agent-oriented-development">agent-oriented development</a>, AEAs are designed to operate in an environment with:

* multiple stakeholders, which
* are represented by AEAs, that
* interact autonomously and

> * communicate _via_ a peer-to-peer network.

Reviewer: Please keep the repetition. It's ok to repeat content on other pages as long as it fits. Here this is helpful as it summarises the environment characteristics.

Author: Done

> * **Representative**: an agent who represents an individual's activities on the Fetch.ai network. An example can be found <a href="../tac-skills">here</a>.

Reviewer: An agent is not a person. Please keep `which` rather than `who` :)

Author: Done
